distribution/Dockerfile

```diff
@@ -60,20 +60,18 @@ RUN apk add --no-cache git \
                         ttf-dejavu \
                         curl \
                         socat \
-                        wget \
-                        nginx \
-                        git # JEP-302
```

Comment:
Also removed `git`, because it was already specified on the L53 above :man_facepalming:.